src/storage/storage.cc

```diff
@@ -641,6 +659,23 @@ Status Storage::ReplicaApplyWriteBatch(std::string &&raw_batch) {
   return Status::OK();
 }
 
+void Storage::RecordStat(StatType type, uint64_t v) {
+  switch (type) {
+    case StatType::FlushCount:
+      db_stats_.flush_count += v;
+      break;
+    case StatType::CompactionCount:
+      db_stats_.compaction_count += v;
+      break;
+    case StatType::KeyspaceHits:
+      db_stats_.keyspace_hits += v;
```

Comment:
Would `fetch_add(relaxed)` better, which might not need to hold the execution pipeline?

Reply:
@mapleFU Yes, it don't require strict sequence increase.